cmd/mcp-standalone/main.go

```diff
@@ -70,12 +72,22 @@ func main() {
 	)
 	tokenClientBuilder.SAToken = token
 
+	k8sMetadata, err := k8s.GetMetadata(k8host, httpClientBuilder.Insecure(true).Build())
+	if err != nil {
+		panic(err)
+	}
+	metadataJsonString, err := json.MarshalIndent(k8sMetadata, "", "  ")
```

Comment:
is this actually needed  now? Prob don't need to print it out at start up?

Reply:
I'll remove. The validation of the response should be enough to ignore whats in it